illumos-utils/src/opte/port_manager.rs

```diff
@@ -931,8 +1002,15 @@ impl PortTicket {
                     "id" => ?&key,
                 );
             }
+        };
+        let mut routes = self.manager.routes.lock().unwrap();
+        remove_key(&mut routes, port.system_router_key());
+        if let Some(key) = port.custom_ipv4_router_key() {
+            remove_key(&mut routes, key);
+        }
+        if let Some(key) = port.custom_ipv6_router_key() {
+            remove_key(&mut routes, key);
         }
```

Comment:
`drop(routes)` here in case someone comes along and adds more stuff after the debug statement?